<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>681</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>5</number>
   </property>
   <widget class="QWidget" name="page_6">
    <widget class="QPushButton" name="logout_button">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>로그아웃</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bcon_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>530</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>입실</string>
     </property>
    </widget>
    <widget class="QPushButton" name="review_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>470</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>수강평</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>입실</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>퇴실</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>200</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>복귀</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>170</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>외출</string>
     </property>
    </widget>
    <widget class="QLabel" name="entrance_line">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>171</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="bcon_button_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>590</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>퇴실</string>
     </property>
    </widget>
    <widget class="QLabel" name="leave_line">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="schedule_check_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>470</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>일정관리</string>
     </property>
    </widget>
    <widget class="QPushButton" name="message_send_button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>470</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>메세지</string>
     </property>
    </widget>
    <widget class="QLabel" name="name_line">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="outing_button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>530</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>외출</string>
     </property>
    </widget>
    <widget class="QPushButton" name="comeback_button">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>590</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>복귀</string>
     </property>
    </widget>
    <widget class="QLabel" name="outing_line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>170</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="comeback_line">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>200</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>530</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_check">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>519</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="login_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>560</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>로그인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="manager_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>관리자모드</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>570</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pw_check">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>560</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="Confirm_line">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>250</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>보내기</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>To. 교수님</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>271</width>
       <height>281</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>370</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>받은메세지</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3"/>
   <widget class="QWidget" name="page_4">
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>직업훈련이력 수강평등록</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>선택</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>훈련 기관명</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>훈련 과정명</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>150</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>201</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>검색</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>290</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>훈련정보</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>371</width>
       <height>111</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Gulim'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;훈련기관 | 대한상공회의소 광주인력개발원&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;훈련과정 | 데이터기반 AI SW 개발자 양성 / 2회&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;훈련구분 | 통합심사과정훈련&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;훈련기간 | 2022.10.25 ~ 2023.05.24&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>440</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>수강후기</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>460</y>
       <width>371</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>620</y>
       <width>371</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>등록하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="main2_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>메인화면</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>401</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="date_line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>390</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="schedule_add_button">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>430</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>추가하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="choice_day">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>380</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="main_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>75</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>메인화면</string>
     </property>
    </widget>
    <widget class="QPushButton" name="schedule_update_button">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>470</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>수정하기</string>
     </property>
    </widget>
    <widget class="QPushButton" name="schedule_del_button">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>510</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>삭제하기</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>15</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>맑은 고딕</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt; 일정관리 &gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="date_detail">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>430</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTableWidget" name="date_list">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>471</y>
       <width>331</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_7">
    <widget class="QPushButton" name="attendance_button">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>출결사항</string>
     </property>
    </widget>
    <widget class="QPushButton" name="schedule_button">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>일정확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="message_button">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>75</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>메세지확인</string>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>411</width>
       <height>591</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_8">
      <widget class="QTableWidget" name="check_widget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>511</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="manager_end_button">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>540</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>관리자모드 종료</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_10">
      <widget class="QTableWidget" name="date_widget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>511</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="manager_end_button_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>540</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>관리자모드 종료</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_9">
      <widget class="QTableWidget" name="message_widget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>381</width>
         <height>511</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="manager_end_button_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>540</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>관리자모드 종료</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
